sig-cli/annual-report-2023.md

```diff
@@ -4,13 +4,15 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-<!--
-   Some example items that might be worth highlighting:
-   - Major KEP advancement
-   - Important initiatives that aren't tracked via KEPs
-   - Paying down significant tech debt
-   - Governance and leadership changes
--->
+    - Interactive delete in kubectl delete
+    - Custom profiling support in kubectl debug
+    - Transition from SPDY to Websockets
+    - Plugin resolution for non-shadowing subcommands in kubectl promoted to beta
+    - New pruning design for kubectl apply --prune, aka ApplySet
+    - Aggregated Discovery promoted to GA
+    - kubectl debug promoted to GA
+    - New minor release for kui
+    - Krew has 242 plugins
 
 2. Are there any areas and/or subprojects that your group needs help with (e.g. fewer than 2 active OWNERS)?
 
```

Comment:
I'd call out here that we've lost 2 kustomize primary maintainers in 2023, we'll be looking for more help in 2024.

Reply:
Updated with this. Thanks for review.